lib/govspeak/editions_extractor.rb

```diff
@@ -0,0 +1,24 @@
+module Govspeak
+  class EditionsExtractor
```

Comment:
@heathd: i've renamed this class to `DependableEditionsExtractor`.. it'll not be appropriate to call it an admin links extractor, because it returns editions. adding dependable to the name indicates that its output contains a specific category of editions. let me know if we can improve it further.